src/main/java/build/buf/protovalidate/exceptions/ValidationException.java

```diff
@@ -16,10 +16,21 @@
 
 /** ValidationException is the base exception for all validation errors. */
 public class ValidationException extends Exception {
+  /**
+   * Creates a new ValidationException with the specified message.
```

Comment:
Super-nit: use a consistent phrasing (right now some places have "Creates a ..." and other places have "Creates a new ...")